### ISSUE TITLE
Doesn't work with screenreaders

### ISSUE THREAD

**Issue by VIPPotato** [WujekFoliarz/DualSenseY-v2#3]
Hello, 
The software doesn't work with screenreaders. This is a huge issue for blind and visually impaired individuals who would like to use it. 
I've tested it with nvda, a free and open source screenreader available at: 
https://www.nvaccess.org/ 
You could also try using it with narrator which is build in to windows. 
The app doesn't read anything, and even object navigation gives up. 
I'm not sure if that's the problem with DSY or dear imgui itself.

**Comment by VIPPotato** [comment]
Is there a possibility to for example use a tolk library and make it output all text to the screenreader? I'm not sure if that's practical though.

**Comment by VIPPotato** [comment]
oh, so there's no way to control the gui from a keyboard? I was thinking of using tab and shift+tab to move between options, up and down arrows to choose values from lists, and space or enter to interact with buttons, switches and maybe a button which would read the status, so UDP connection, controller battery etc. I'm not able to use a mouse properly.

**Comment by WujekFoliarz** [comment]
I guess there is since I've seen it in other imgui programs

**Comment by VIPPotato** [comment]
Can you try to implement it then please? It would be really great considering other programs with similar functionality including dsx are hard to use with a screenreader. I'm not sure how to work on it codewise, so I only can provide testing and feedback. Thanks in advance.
